lib/presentation/view/home.dart

```diff
@@ -1,15 +1,237 @@
+                decoration: const BoxDecoration(
+                  border: BorderDirectional(
+                    bottom: BorderSide(
+                      color: Colors.black54,
+                    ),
+                  ),
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    const Text(
+                      TextConstants.randomRecipesTitle,
+                      style: TextStyle(
+                        fontSize: Dimensions.bigFontSize,
+                        fontWeight: FontWeight.w400,
+                        color: Colors.black54,
+                      ),
+                    ),
+                    GestureDetector(
+                      key: Keys.refreshRecipes,
+                      onTap: () => recipesController
+                          .getRandomRecipes(ApiConstants.numberOfRandomRecipes),
+                      child: const Icon(
+                        Icons.refresh,
+                        color: Colors.black54,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+          recipesController.obx(
+            (state) {
+              return SliverPadding(
+                padding: EdgeInsets.symmetric(
+                  horizontal: MediaQuery.of(context).size.width *
+                      Dimensions.fivePercent,
+                ),
+                sliver: SliverLayoutBuilder(
+                  builder:
+                      (BuildContext context, SliverConstraints constraints) {
+                    if (constraints.crossAxisExtent >= Dimensions.bigScreen) {
+                      return SliverList(
+                        delegate: SliverChildBuilderDelegate(
+                          (BuildContext context, int index) {
+                            return Builder(
+                              builder: (context) {
+                                return Row(
+                                  children: [
+                                    Flexible(
+                                      child: RecipeCard(
+                                        recipe: state![index * 2],
+                                        cardIndex: index * 2,
+                                      ),
+                                    ),
+                                    ((index * 2) + 1 < state.length)
+                                        ? Flexible(
+                                            child: RecipeCard(
+                                              recipe: state[(index * 2) + 1],
+                                              cardIndex: (index * 2) + 1,
+                                            ),
+                                          )
+                                        : Flexible(
+                                            child: Container(),
+                                          )
```

Comment:
Here you can replace it with const SizedBox.shrink

Reply:
I tried to change it but I can't make it a constant